ejb/src/main/resources/schema/jboss-ejb3-4_0.xsd

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ JBoss, Home of Professional Open Source
+~ Copyright 2019, Red Hat, Inc., and individual contributors as indicated
+~ by the @authors tag.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="https://jakarta.ee/xml/ns/jakartaee"
+           xmlns="http://www.jboss.com/xml/ns/jakartaee"
+           xmlns:javaee="https://jakarta.ee/xml/ns/jakartaee"
```

Comment:
Not sure about the overall namespace naming in this project, but a new name like `jakartaee` seems more natural, to be distinct from the legacy name.

Reply:
ok